exercises/concept/community-garden/test/community_garden_test.exs

```diff
@@ -45,6 +45,32 @@ defmodule CommunityGardenTest do
     assert plot_3.plot_id == 3
   end
 
+  @tag task_id: 3
+  test "registered plots have incremental unique id when registered concurrently" do
+    {:ok, pid} = CommunityGarden.start()
+
+    total_plots = 20
+    test_process_pid = self()
+
+    Enum.each(1..total_plots, fn n ->
+      spawn(fn ->
+        plot = CommunityGarden.register(pid, "Mary Bumblebee #{n}")
+        send(test_process_pid, {n, plot})
+      end)
+    end)
+
+    plot_ids =
+      Enum.map(1..total_plots, fn n ->
+        receive do
+          {^n, plot} -> plot.plot_id
+        after
+          100 -> nil
```

Comment:
20 * 100ms = 2s, so even is somebody messes up their solution so bad they trigger the timeout every time, they should still see some output of the test. I think the test runner only gets killed by exercism after 3 * average_run_time (which is 3 * 4s in our case)